hermes/file_gen.go

```diff
@@ -0,0 +1,42 @@
+package main
+import (
+	"fmt"
+	// "math/rand"
+	"strconv"
+	"crypto/sha1"
+)
+type HermesFile struct {
+	name string
+	contents string
+}
+func generate_file_name (file_ID int, file_checksum string) string{
+	var id_string string=""
+	if file_ID < 10 {
+		id_string = "0" + strconv.Itoa(file_ID)
+	} else {
+		id_string = strconv.Itoa(file_ID)
+	}
+	var file_name string = "Hermes_"+id_string+file_checksum
```

Comment:
Using := instead of var = is more consise.

```suggestion
	file_name string := "Hermes_" + id_string + file_checksum
```